src/models/layerMetadata/layer3DMetadata.ts

```diff
@@ -1032,7 +980,6 @@ export class Layer3DMetadata implements ILayer3DMetadata, IMetadataCommonModel {
   @fieldConfig({
     category: FieldCategory.GENERAL,
     isManuallyEditable: true,
-    isAutoGenerated: true,
```

Comment:
why was it removes?

Reply:
Because producerName is not auto generated